ckcp/openshift_dev_setup.sh

```diff
@@ -178,12 +178,33 @@ install_cert_manager() {
     argocd app wait "$APP" >/dev/null 2>&1
   fi
 
-  # Wait until cert-manager is ready
+  # Wait until cert-manager pods are ready
   pods_cmd="kubectl -n openshift-cert-manager get pods -o=jsonpath='{range .items[*]}{.metadata.name}{\" \"}{.status.containerStatuses[*].ready}{\"\n\"}{end}'"
   until [[ $(eval $pods_cmd | wc -l) -eq 3 ]]; do
     echo -n "."
     sleep 5
   done
+
+  # perform a dry-run create of a cert-manager
+  # Certificate resource in order to verify that CRDs are installed and all the
+  # equired webhooks are reachable by the K8S API server.
+  local tmp_dir=$(mktemp -d)
+  echo "
```

Comment:
I'm not a big fan of in-lining resources in scripts. Maybe you could create the yaml in `ckcp/openshift-operators/cert-manager/`? That would get rid of the tmp_dir as well.

Reply:
The current in-lining resource of  is more like a temporary  resource, it's useless for any applications. It doens't make sense to create the yaml in  ckcp/openshift-operators/cert-manager/

Reply:
Then the tmp_dir should be cleaned up .

Reply:
@Roming22 I would like to use this file `ckcp/openshift/base/certs.yaml` for executing dry-run check. WDYT? So  we don't need the in-lining resources in scripts

Reply:
fixed.